src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -76,20 +94,36 @@ public function __construct(
      * @Transform /^"([^"]+)" product$/
      * @Transform :product
      */
-    public function castProductNameToProduct($productName)
+    public function getProductByName($productName)
     {
         $product = $this->productRepository->findOneBy(['name' => $productName]);
         if (null === $product) {
-            throw new \InvalidArgumentException('Product with name "'.$productName.'" does not exist');
+            throw new \InvalidArgumentException(sprintf('Product with name "%s" does not exist', $productName));
         }
 
         return $product;
     }
 
     /**
+     * @Transform /^"([^"]+)" variant of product "([^"]+)"$/
+     */
+    public function getProductVariantByNameAndProduct($variantName, $productName)
+    {
+        $product = $this->getProductByName($productName);
+
+        $productVariant = $this->productVariantRepository->findOneBy(['presentation' => $variantName, 'object' => $product]);
+        if (null === $productVariant) {
+            throw new \InvalidArgumentException(sprintf('Product variant with name "%s" of product "%s" does not exist', $variantName, $productName));
+        }
+
+        return $productVariant;
+    }
+
+    /**
+     * @Given /^store has a product "([^"]+)"$/
      * @Given /^store has a product "([^"]+)" priced at "(?:€|£|\$)([^"]+)"$/
      */
-    public function storeHasAProductPricedAt($productName, $price)
+    public function storeHasAProductPricedAt($productName, $price = '0.00')
```

Comment:
According to @michalmarcinkowski [comment](https://github.com/Sylius/Sylius/pull/3915#discussion_r52173045) we should set product price to ``0.00`` if not provided. So product default price value is ``0.00`` not ``null`` :) moreover, we pass ``$price`` to ``getPriceFromString`` method and if it would be null, we should check it there.